lib/falcon/service/server.rb

```diff
@@ -43,7 +43,10 @@ def start
 				
 				preload!
 				
-				Console.logger.info(self) {"Starting #{self.name} on #{@endpoint}"}
+				Console.logger.info(self) do
+					"Starting #{self.name} on #{@endpoint}"
+					"- Using certificate: #{Localhost::Authority.path}" if @endpoint.secure?
```

Comment:
I'm not sure this is always the correct path, as the path can be configured by the service environment. Can you check how it works and update the PR?

Reply:
That was fast! Thank you. :)

Correct, it's not, I was just final checking and noticed I was sloppy.  Which is why I closed it (before you commented) to continue investigating.  It's the directory, not the certificate within.

What I want is the Localhost::Authority instance, for `authority.certificate_path`.  But it isn't included in the [built SSL context](https://github.com/socketry/falcon/blob/31241cf8df2108a33a9ec23fd7b10dabea8e80b2/lib/falcon/endpoint.rb#L26), only the authority's _server context_.

I'm looking to see if the authority instance is elsewhere.

Reply:
Ah right, in general I don't think we can extract the certificate path from the SSL context.

Maybe we can log it from the environment configuration? But it'a also not guaranteed. Maybe the environment should be like "Here are some useful keys for you to log".

Reply:
Yes, it seems the OpenSSL X509 Certificate object doesn't keep track of the filename after `.load_file(...)`, kind of a shame.  I'm reading through the environment-related code.